react/features/rtcstats/RTCStats.ts

```diff
@@ -121,16 +120,6 @@ class RTCStats {
         this.trace && this.trace.statsEntry('dominantSpeaker', null, dominantSpeakerData);
     }
 
-    /**
-     * Send e2e rtt data, the data will be processed by rtcstats-server and saved in the dump file.
-     *
-     * @param {Object} e2eRttData - The object that holds the e2e data.
-     * @returns {void}
-     */
-    sendE2eRttData(e2eRttData: E2ERTTData) {
```

Comment:
@mykhailo-ua I think we want to keep the analytics and stats part, just the UI needs to go away.

Reply:
reverted that part.